buildpack/infrastructure/storage.py

```diff
@@ -170,7 +170,7 @@ def _runtime_sts_support(version):
     elif (
         version >= 9.2
         or (version.major == 8 and version >= MXVersion("8.18.7"))
-        or (version.major == 7 and version >= MXVersion("7.23.22"))
+        or (version.major == 7 and version >= MXVersion("7.23.30"))
```

Comment:
This check should not change.

The previous `if` statement where we check for `>= 9.6.1` and `>= 8.18.11` requires an additional statement where we check for `>= 7.23.30` as well.

Reply:
I did the changes, could you please check that.